ouster-ros/CMakeLists.txt

```diff
@@ -46,6 +46,7 @@ set(BUILD_SHARED_LIBS OFF)
 
 option(BUILD_VIZ "Turn off building VIZ" OFF)
 option(BUILD_PCAP "Turn off building PCAP" OFF)
+option(BUILD_OSF "Turn off building OSF" OFF)
```

Comment:
Out of scope, but the language on these comments is confusing. The wording implies that when set to OFF it builds these options.